src/components/Swap/Output.tsx

```diff
@@ -85,7 +85,7 @@ export default function Output({ disabled, focused, children }: PropsWithChildre
           <ThemedText.Body2 color="secondary" userSelect>
             <Row>
               <USDC gap={0.5} isLoading={isRouteLoading}>
-                {outputUSDC ? `$${formatCurrencyAmount(outputUSDC, 6, 'en', 2)}` : '-'}{' '}
+                {outputUSDC && `$${formatCurrencyAmount(outputUSDC, 6, 'en', 2)} `}
```

Comment:
The space at the end of the string literal is intentional, right? I _think_ it's necessary, just double-checking.

Reply:
yep! intentional